web/_js/main/infoblock.js

```diff
@@ -158,10 +158,78 @@ function createInfoBlock(entry, mode = 0) {
 		idElementContainer.appendChild(editElement)
 	}
 
+	// Adds giscus button
+	if (entry.id >= 0) {
+		const giscusButton = createGiscusButton(entry)
+		idElementContainer.appendChild(giscusButton)
+	}
+
 	// Removes empty elements
 	if (!bodyElement.hasChildNodes()) bodyElement.remove()
 	if (!linkListElement.hasChildNodes()) linkListElement.remove()
 	if (!listElement.hasChildNodes()) listElement.remove()
 
 	return element
+}
+
+function createGiscusButton(entry) {
+	const button = document.createElement("button")
+	button.className = "btn btn-sm btn-outline-primary"
+	button.innerHTML = '<i class="bi bi-chat-dots" aria-hidden="true"></i> Comments'
```

Comment:
`bi-chat-dots` should be changed to `bi-chat-dots-fill` to match the existing visual style of the filled pencil edit button icon.

![image](https://github.com/user-attachments/assets/91eee30d-1953-47fe-9ee3-e159afa85ad1)